src/map/clif.c

```diff
@@ -1065,6 +1065,7 @@ void clif_set_unit_idle(struct block_list* bl, struct map_session_data *tsd, enu
 #endif
 #if PACKETVER >= 20150513
 	p.body = vd->body_style;
+	memcpy(p.name, status->get_name(bl), NAME_LENGTH);
```

Comment:
status->get_name() not have target sd or bl paramerter. Because this do any actions with name in get_name() function is useless.
This need because now in one packet combined two old packets. With old packets function can be hooked. But here need hook all spawn/move functions for change only name what was before in one separate packet. I suggest move it strsafecpy with name to separate function.